src/test/java/UhrzeitenTest.java

```diff
@@ -18,23 +17,10 @@ public void assembleAllValidTimes() {
 		assertThat(times.size(), not(2000));
 	}
 
-	@Test(expected = IndexOutOfBoundsException.class)
+	@Test(expectedExceptions = IndexOutOfBoundsException.class)
 	public void testIndexOutOfBoundsException() {
 		Uhrzeiten uhrzeiten = new Uhrzeiten();
 		List<String> times = uhrzeiten.assembleAllValidTimes();
 		times.get(-1);
 	}
-
-	@Test
```

Comment:
I think you'll get your chance. Did some changes in between your new ones and now my pulled branch can't be merged - i guess because of the new naming of the folder. I bet I shoud have rebased your changes before committing. I havent figured out yet how to deal with that...